common/src/main/java/org/apache/rocketmq/common/message/MessageClientIDSetter.java

```diff
@@ -48,7 +50,7 @@
     }
 
     private synchronized static void setStartTime(long millis) {
-        Calendar cal = Calendar.getInstance();
+        Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("GMT+8"));
```

Comment:
It is recommended to use the time zone as an optional option instead of the GMT+8 by default